src/TitleDetails/BoundPiecesList/constants.js

```diff
@@ -0,0 +1,36 @@
+import {
+  getItemStatusLabel,
+  ITEM_STATUS,
+} from '@folio/stripes-acq-components';
+import { NoValue, TextLink } from '@folio/stripes/components';
+
+import { PIECE_COLUMNS } from '../constants';
+
+export const BOUND_ITEMS_LIMIT = 20;
+export const VISIBLE_COLUMNS = [
+  PIECE_COLUMNS.barcode,
+  PIECE_COLUMNS.itemStatus,
+  PIECE_COLUMNS.displaySummary,
+  PIECE_COLUMNS.callNumber,
+];
+
+export const COLUMN_FORMATTER = (hasViewInventoryPermissions, instanceId) => {
```

Comment:
Let's name functions in a common way, e.g. `getColumnFormatter` or smth like this.